Sources/JobsPostgres/RetentionPolicy.swift

```diff
@@ -0,0 +1,41 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Hummingbird server framework project
+//
+// Copyright (c) 2025 the Hummingbird authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See hummingbird/CONTRIBUTORS.txt for the list of Hummingbird authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+
+/// Data rentension policy
+public struct RetentionPolicy: Sendable {
+    /// Data retention
+    public struct RetainData: Sendable {
```

Comment:
Maybe this could be
```swift
enum RetainData {
    case retain(for: Duration)
    case doNotRetain
}
```
Anything with doNotRetain gets deleted immediately